src/main/java/net/dv8tion/jda/internal/entities/UserImpl.java

```diff
@@ -73,6 +74,28 @@ public String getAvatarId()
         return avatarId;
     }
 
+    @Nonnull
+    @Override
+    public RestAction<Profile> retrieveProfile()
+    {
+        return new DeferredRestAction<>(getJDA(), Profile.class, this::getProfile, () -> {
+            Route.CompiledRoute route = Route.Users.GET_USER.compile(getId());
+            return new RestActionImpl<>(getJDA(), route, (response, request) -> {
+                DataObject json = response.getObject();
+
+                String bannerId = json.getString("banner", null);
+                int accentColor = json.getInt("accent_color", User.DEFAULT_ACCENT_COLOR_RAW);
+
+                return new Profile(getIdLong(), bannerId, accentColor);
+            });
+        });
+    }
+
+    public Profile getProfile()
+    {
+        return profile;
+    }
```

Comment:
We don't actually expose `getProfile` on `User` anymore right? Why is this still here?